devtools/debugging_an_entry_point.py

```diff
@@ -0,0 +1,39 @@
+"""Test an entry point.
+
+This script is used to debug the entry point function imported and run below. This
+function expecting to be provided with values via command line arguments and argparse,
+and so debugging requires a way to pass those arguments in to the function.
+
+Within Visual Studio Code, a new debug configuration can be added to .vscode/launch.json
+containing the required arguments, which will then be passed into the script as it runs.
+The JSON to do that looks like this:
+
+```json
+    {
+        "name": "_safedata_validator_cli debug",
+        "type": "python",
+        "request": "launch",
+        "program": "${file}",
+        "console": "integratedTerminal",
+        "args": [
+            "-r",
+            "local/config_files/live_safeweb_gbif_2016_07_25.cfg",
```

Comment:
I think it would help to explain what these files are, as I guess that they will be changed based on what is being debugged and for different users?